src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/EditProjectFileCommand.cs

```diff
@@ -115,22 +147,26 @@ private void TextDocument_FileActionOccurred(object sender, TextDocumentFileActi
             }));
         }
 
-        private async Task<string> GetFileAsync(string projectFileName)
+        private async Task SetupFileAsync()
+        {
+            _tempProjectPath = GetTempFileName(Path.GetFileName(_unconfiguredProject.FullPath));
```

Comment:
What if the project is renamed? You've got lots of state that caches file names.